composer.json

```diff
@@ -33,7 +33,7 @@
     "squizlabs/php_codesniffer": "^3.3.2",
     "symfony/console": "^4.0",
     "symfony/framework-bundle": "^4.0",
-    "symfony/http-foundation": "^4.0",
+    "symfony/http-foundation": "^4.0|^5.0",
```

Comment:
Why this change? Doesn't seem related to the rest.

Reply:
The InputBag is new in Symfony 5, didn't exist in 4 yet, so to run the tests we need Symfony 5

Reply:
There currently isn't a nice way to test it in a 3rd party package, so you can delete the file `tests/Type/Symfony/input_bag_get.php` which isn't very useful currently.

Reply:
Okay, fair, so just delete the tests altogether?

Reply:
Oh sorry, I haven't noticed there's actually a way here in phpstan-symfony with the ExtensionTestCase :) I just thought you create the file `input_bag_get.php` and let PHPStan analyse it, but right now you're actually checking the return types which is great :) So feel free to add some tests that fail with `mixed`, `string|null` etc. and see how my suggested logic fixes that :)

Reply:
I'm not quite sure what you want me too add there? The `$default` parameter can technically be mixed, however, anything other than a string will throw a deprecation notice. Besides, falling back on default will say `string|null` either way, no matter what you put in `$default`, so I'm not sure if it would matter. I'm fine with adding other types to the test but the result will be the same :P

Reply:
`mixed` means that it can be a string or null and we can't be sure, so the return type also has to be `string|null`. I just want to test the extension for `mixed` and `string|null` types as the second parameter.

Reply:
PR updated, I think that's what you were looking for?

Reply:
Approximately :) Thanks.

Reply:
Please fix the build: https://travis-ci.com/github/phpstan/phpstan-symfony/jobs/372130355